### DISCUSSION TITLE
Is it possible to automatically add an ephemeral node to a room?

### DISCUSSION THREAD

**Discussion by JWambaugh** [netdata/netdata#15869] [Q&A] [answered]
We are using netdata cloud, and we use autoscaling extensively in our infrastructure, so are dealing with ephemeral nodes that get started and shut down as traffic demands.
I have a parent server setup that forwards all metrics from the ephemeral nodes to netdata cloud.

I would like to organize the ephemeral nodes by room so I can see all the nodes responsible for a specific website. Is there a way to automatically add a new node to a room? Manually adding nodes to a room in the UI isn't going to work when we have nodes starting up and shutting down throughout the day.

**Comment by andrewm4894** [comment]
Hello @JWambaugh 

This is for sure possible. Each room has a different room id, you can claim nodes to multiple rooms on a space via install or command line and so no need go near the UI.

Let me find docs and some examples...

**Comment by andrewm4894** [comment, marked as answer]
So if i wanted to claim a node to two different rooms i could do this:

```
wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh --stable-channel --claim-token XXX --claim-rooms ae33f57b-b54e-4236-a6de-054da3f0a748,3d404df0-de8c-484e-b74d-1c0d0dcfedee --claim-url https://app.netdata.cloud
```

just pass in the `--claim-rooms` as a list of room id's in the same space (node can only be claimed to one space but can be in multiple rooms within the space) like above.

So if using the [helm chart](https://github.com/netdata/helmchart) for example its the `parent.claiming.room` and `child.claiming.room` where you could pass in the list of room id's you can get from Netdata Cloud. 

You can get room id from the UI like this:

![image](https://github.com/netdata/netdata/assets/2178292/2caa9a0a-4dfb-40df-b447-0fb800dae29a)

**Reply by JWambaugh** [reply]
Awesome, thank you!

**Comment by andrewm4894** [comment]
Just leaving here for reference too - if using something like Ansible or some other configuration management tool you can also do [something like this](https://github.com/netdata/community/blob/main/configuration-management/ansible-ml-demo/tasks/netdata/claim.yaml) using the claim script itself too.
